commit-message-hook.sh

```diff
@@ -7,9 +7,15 @@ fi
 # Include any branches for which you wish to disable this script
   BRANCHES_TO_SKIP=(master develop staging test)
 
+#Error message to be returned if branch name is not named properly
+ERROR="Branch name is not named properly, please see wiki for formating: https://github.com/space-concordia-robotics/robotics-prototype/wiki/Work-Flow"
+
 # Get the current branch name
 BRANCH_NAME=$(git symbolic-ref --short HEAD)
 
+# Trims down the branch name to not include the issue number.
+TRIM_NUM=$(echo $BRANCH_NAME | sed -e 's/[0-9]//g' )
```

Comment:
What happens if we have a branch of the type `ubuntu-18-integration-206` ? It will also remove the 18? no?

I haven't done regex in a while but it should look like this "-[0-9]*$" (Hyphen followed by any number of 0 to 9 and ends with end of string($).

Reply:
TRIM_NUM is used to check if a branch name ends with a hyphen after the issue number, so this won't affect the name verification check.